decidim-core/lib/decidim/content_parsers/user_parser.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ContentParsers
+    class UserParser < BaseParser
+      MENTION_REGEX = /(^|\s)@([a-z0-9]\w*)/
```

Comment:
Are underscores allowed as a valid nickname character?

Reply:
Yes, the `\w` is a shortcut for `[a-zA-Z0-9_]`, but I will modify it a bit to also capture nicknames starting with a uppercase letter. So, the nickname can start with a letter or number, but not with a underscore. The final regex will be `(^|\s)@([a-zA-Z0-9]\w*)`